src/components/Quiz/QuizzesStats.tsx

```diff
@@ -0,0 +1,193 @@
+          >
+            <Button
+              variant="outline-color"
+              leftIcon={<Icon as={FaTwitter} />}
+              onClick={() =>
+                handleShare({ score: userScore, total: TOTAL_QUIZZES_POINTS })
+              }
+              w={{ base: "full", lg: "auto" }}
+              mt={{ base: 2, lg: 0 }}
+            >
+              <Translation id="share-results" />
+            </Button>
+          </GridItem>
+
+          <GridItem colSpan={2} order={{ base: 2, lg: 3 }}>
+            <Stack gap={2}>
+              <Flex
+                justifyContent={{ base: "center", lg: "flex-start" }}
+                alignItems="center"
+              >
+                <Circle size="64px" bg="primary" mr={4}>
+                  <TrophyIcon color="neutral" w="35.62px" h="35.62px" />
+                </Circle>
+
+                <Text fontWeight="bold" fontSize="5xl" mb={0} color="body">
+                  {userScore}
+                  <Text as="span" color="bodyLight">
+                    /{TOTAL_QUIZZES_POINTS}
+                  </Text>
+                </Text>
+              </Flex>
+
+              <Progress value={userScore} />
+
+              <Flex direction={{ base: "column", lg: "row" }}>
+                <Text mr={10} mb={0} mt={{ base: 2, lg: 0 }} color="bodyLight">
+                  <Translation id="average-score" />{" "}
+                  <Text as="span" color="body">
+                    {average}%
+                  </Text>
+                </Text>
+
+                <Text mb={0} color="bodyLight">
+                  <Translation id="completed" />{" "}
+                  <Text as="span" color="body">
+                    {numberOfCompletedQuizzes}/{totalQuizzesNumber}
+                  </Text>
+                </Text>
+              </Flex>
+            </Stack>
+          </GridItem>
+        </Grid>
+
+        {/* community stats */}
+        <Flex
+          direction="column"
+          gap={6}
+          justifyContent="space-between"
+          bg="ednBackground"
+          borderRadius={{ base: "none", lg: "lg" }}
+          border="none"
+          p={{ base: 8, lg: 12 }}
+        >
+          <Text fontWeight="bold" fontSize="xl" mb={0}>
+            <Translation id="community-stats" />
+          </Text>
+
+          <Flex
+            direction={{ base: "column", md: "row" }}
+            gap={{ base: 6, md: 10 }}
+          >
+            <Stack>
+              <Text mr={10} mb={-2} color="bodyLight">
+                <Translation id="average-score" />
+              </Text>
+              {/* Data from Matomo, manually updated */}
+              <Text color="body">67.4%</Text>
+            </Stack>
+
+            <Stack>
+              <Text mr={10} mb={-2} color="bodyLight">
+                <Translation id="questions-answered" />
+              </Text>
+
+              {/* Data from Matomo, manually updated */}
+              <Text color="body">100 000+</Text>
```

Comment:
Should we localise number formatting here? If not, we should use a comma separator (100,000+)

```
{new Intl.NumberFormat(localeForStatsBoxNumbers).format(number)}
```

Reply:
As discussed @nhsz, added here: https://github.com/ethereum/ethereum-org-website/pull/10349

Reply:
Fixed now